CMakeLists.txt

```diff
@@ -404,7 +404,7 @@ IF( WIN32 )
         ${QtQuick_location}
         ${QtPositioning_location}
    )
-   IF( NOT ${NO_QTMULTIMEDIA})
+   IF(${NO_QTMULTIMEDIA})
       SET( Qt_DLLs,
         ${Qt_DLLs}
         ${QtMultimedia_location}
```

Comment:
This is adding QtMultimedia DLLs. I think we should be adding these DLLs unless NO_QTMULTIMEDIA is ON, right?